crates/teloxide/examples/chat_member_updates.rs

```diff
@@ -0,0 +1,92 @@
+// Import Teloxide and dependencies modules
+use teloxide::prelude::*;
+use teloxide::types::{ParseMode, ParseMode::MarkdownV2, ChatMemberUpdated, Update};
+use teloxide::dispatching::Dispatcher;
+use teloxide_core::adaptors::DefaultParseMode;
+
+// We create a type alias for our bot type
+type Bot = DefaultParseMode<teloxide::Bot>;
+
+// Main function that starts the bot
+#[tokio::main]
+async fn main() -> ResponseResult<()> {
+
+    // Initialize the logging environment for Teloxide
+    pretty_env_logger::init();
+
+    // Print a message to the console
+    log::info!("Starting Welcome/Goodbye bot...");
+
+    // in this case we use env variables to get the bot token
+    let bot = teloxide::Bot::from_env().parse_mode(MarkdownV2); // We use parse_mode method for use MarkdownV2 by default
+
+    // We create a handler for our botc
+    let handler = dptree::entry().inspect(|u:Update| {
+        println!("{u:#?}"); // Print the update to the console with inspect method and a closure
+    })
```

Comment:
It's a good idea to teach about `inspect`, but I'm not sure if printing update makes sense here, this is not something you want to do in real bots...

Reply:
We could change the comment explaining that we printed the update for debugging purposes.